docs/clients/ledger.md

```diff
@@ -0,0 +1,27 @@
+# Ledger // Cosmos
+
+### Ledger Support for Account Keys
+
+`gaiacli` now supports derivation of account keys from a Ledger seed. To use this functionality you will need the following:
```

Comment:
Let's reference the BIP HD derivation spec, which we use - https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki

Reply:
Added a note below about this with links to our derivation paths in the code.